docs/lsm/allocation/allocation_sources/allocation_v3/basic_example/v3_plugin.py

```diff
@@ -0,0 +1,15 @@
+@allocation_helpers.allocator()
+def get_top_level_value(
+    service: "lsm::ServiceEntity",
+    attribute_path: "string",
+) -> "int":
+    return 1
+
+@allocation_helpers.allocator()
+def get_embedded_value(
+    service: "lsm::ServiceEntity",
+    attribute_path: "string",
+) -> "int":
+    return 3
```

Comment:
It would be nice to illustrate more what these allocators do, and when they run.  Something like
```python
@allocation_helpers.allocator()
def get_value(
    service: "lsm::ServiceEntity",
    attribute_path: "string",
    *,
    value: "any",
) -> "any":
    """
    Store a given value in the attributes of a service.

    :param value: The value to store on the service.
    """
    print("This will be called in the first validation compile only!", str(service), str(attribute_path), str(value))
    return value
```

And then you can call this for both the top level value and the embedded value